org.gnome.Hamster.yaml

```diff
@@ -0,0 +1,52 @@
+---
+app-id: org.gnome.Hamster
```

Comment:
It this a GNOME hosted project ?

(changing the app-id has consquences all down the way, including with DBus, so I'm not requesting it)

Reply:
I'm not sure, I used the same ID in the XML they have in there own repo.

Reply:
Yeah this is one of those older apps from before GNOME Circle etc.